alicloud/data_source_alicloud_rds_instances.go

```diff
@@ -0,0 +1,289 @@
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"output_file": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+
+			// Computed values
+			"instances": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"pay_type": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"db_instance_type": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"region_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"create_time": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"expire_time": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"db_instance_status": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"engine": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"engine_version": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"db_instance_net_type": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"connection_mode": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"lock_mode": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"lock_reason": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"db_instance_class": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"instance_network_type": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"vpc_cloud_instance_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"zone_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"multi_or_single": {
+							Type:     schema.TypeBool,
+							Computed: true,
+						},
+						"master_instance_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"guard_db_instance_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"temp_db_instance_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"readonly_db_instance_ids": {
+							Type:     schema.TypeList,
+							Elem:     &schema.Schema{Type: schema.TypeString},
+							Computed: true,
+						},
+						"vpc_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"vswitch_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"replicate_id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"resource_group_id": {
+							Type:     schema.TypeString,
+							Computed: true,
```

Comment:
There is no need to show 'replicate_id' and 'resource_group_id', and please remove them.

Reply:
removed